v1/plugin/meta.go

```diff
@@ -85,6 +85,7 @@ const (
 	collectorType pluginType = iota
 	processorType
 	publisherType
+	streamCollectorType
```

Comment:
Consider not including it in this PR, but rather open a new one to address that.